includes/llms.functions.core.php

```diff
@@ -359,7 +360,7 @@ function llms_get_dom_document( $string ) {
 	$libxml_state = libxml_use_internal_errors( true );
 
 	$dom = new DOMDocument();
-	if ( ! $dom->loadHTML( mb_convert_encoding( $string, 'HTML-ENTITIES', 'UTF-8' ) ) ) {
+	if ( ! $dom->loadHTML( '<meta http-equiv="Content-Type" content="text/html; charset=utf-8">' . $string ) ) {
```

Comment:
can we turn this into:
```suggestion
    $string = $skip_utf8_encoding_conversion ? $string : string . '<meta http-equiv="Content-Type" content="text/html; charset=utf-8">' . $string;
	if ( ! $dom->loadHTML( $string ) ) {
```

where `$skip_utf8_encoding_conversion` is an additional parameter of this function set to `false` by default, so that when certificates calls this function it passes `true` as second parameter.
Otherwise we have an invalid html code:
`Error: A document must not include both a meta element with an http-equiv attribute whose value is content-type, and a meta element with a charset attribute.`
source https://validator.w3.org/nu/#textarea when pasting there the result of a certificate export.

This because when exporting a certificate we export the whole html which already contains a `<meta charset="UTF-8">`.

Reply:
I was wrong on this, as this is actually what does the magic:
`'<meta http-equiv="Content-Type" content="text/html; charset=utf-8">'`

Reply:
hmmm hmmm mmm hmmm....

puzzled. Ill take another stab in a bit

Reply:
@eri-trabiccolo I was able to fix the markup error by removing the meta element afterwards.

It looks like as long as this is present when the HTML is loaded, it doesn't matter if it's removed when `saveHTML()` is called (which is where we'll see that invalid markup).

The tests were failing when you were working on this because with the removal it actually works the way it would have when using `mb_convert_encoding()` -- the tests were setup to check against the actual characters but with this method now it converts the characters to html entity codes.

The tests have been adjusted to accommodate and I think this is now better than what I initially submitted.